### PR TITLE
Add test for passing thrown error in onError

### PR DESCRIPTION
**Pull request by restrry**

### CODE REVIEW

File: packages/core/test/utils/createMockTask.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      assert.equal(
+        task.isRunning(),
+        false,
+        'Sagas must take consecutive actions dispatched synchronously on an action channel even if it performs blocking calls',
+      )
+      assert.end()
+    })
+    .catch(err => assert.fail(err))
+
+  // const generator = saga()
```
Comment (shinima) [inline packages/core/test/utils/createMockTask.js]: Could you delete these comments?

Comment (restrry) [reply packages/core/test/utils/createMockTask.js]: oops

File: packages/core/test/interpreter/onerror.js [hunk 2]
```diff
@@ -26,6 +26,52 @@ test('saga onError is optional', assert => {
   })
 })
 
+test('saga passes thrown Error instance in onError handler', assert => {
```
Comment (Andarist) [inline packages/core/test/interpreter/onerror.js]: isnt this test a duplicate of "saga onError is called for uncaught error"?

Comment (restrry) [reply packages/core/test/interpreter/onerror.js]: didn't notice 👍 
changed test case

**Pull request merged**